src/99-appendices/07-meg-systems.md

```diff
@@ -8,13 +8,13 @@ Restricted keywords for Manufacturer field in the `*_meg.json` file:
 -   [`KIT/Yokogawa/Ricoh`](06-meg-file-formats.md#kityokogawaricoh)
 -   [`KRISS`](06-meg-file-formats.md#kriss)
 -   [`ITAB`](06-meg-file-formats.md#itab)
--   [`Aalto/MEG–MRI`](06-meg-file-formats.md#aalto-megmri)
+-   [`Aalto/MEG/MRI`](06-meg-file-formats.md#aalto-megmri)
```

Comment:
Why did you perform this change? I am afraid we cannot change this restricted keyword.

Reply:
resolved here https://github.com/bids-standard/bids-specification/pull/400/files/1ea328ea38dc07811c4139c70bec13a5a2feb93e#r381360490